tests/cli/test_build_and_deploy_aws.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Make sure we can build an image and deploy it inside AWS!
+#
+#####
+
+. /usr/share/beakerlib/beakerlib.sh
+
+CLI="./src/bin/composer-cli"
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        if [ -z "$AWS_ACCESS_KEY_ID" ]; then
+            rlFail "AWS_ACCESS_KEY_ID is empty!"
+        else
+            rlLogInfo "AWS_ACCESS_KEY_ID is configured"
+        fi
+
+        if [ -z "$AWS_SECRET_ACCESS_KEY" ]; then
+            rlFail "AWS_SECRET_ACCESS_KEY is empty!"
+        else
+            rlLogInfo "AWS_SECRET_ACCESS_KEY is configured"
+        fi
+
+        AWS_BUCKET="${AWS_BUCKET:-composerredhat}"
+        AWS_REGION="${AWS_REGION:-us-east-1}"
+
+        rlLogInfo "AWS_BUCKET=$AWS_BUCKET"
+        rlLogInfo "AWS_REGION=$AWS_REGION"
+
+        if ! rlCheckRpm "python3-pip"; then
+            rlRun -t -c "dnf -y install python3-pip"
+            rlAssertRpm python3-pip
+        fi
+
+        rlRun -t -c "pip3 install awscli"
+
+        # aws configure
+        [ -d ~/.aws/ ] || mkdir ~/.aws/
+
+        if [ -f ~/.aws/config ]; then
+            rlLogInfo "Reusing existing ~/.aws/config"
+            rlRun -t -c "cat ~/.aws/config"
```

Comment:
Does the cat really need to be there? Or was that just for debugging?

Reply:
More for debugging and easy access to information from the test log. However if you think we can expose information in this way I can remove it. My understanding is that `.aws/config` doesn't contain anything like access credentials and passwords.

Reply:
I think it's better to remove it, you can put secrets in there and it'd be pretty bad for them to end up in logs.